macros/union/github_union_connections.sql

```diff
@@ -0,0 +1,94 @@
+{% macro github_union_connections(connection_dictionary, single_source_name, single_table_name, default_identifier=single_table_name) %}
+
+{{ adapter.dispatch('github_union_connections', 'github') (connection_dictionary, single_source_name, single_table_name, default_identifier) }}
+
+{%- endmacro %}
+
+{% macro default__github_union_connections(connection_dictionary, single_source_name, single_table_name, default_identifier=single_table_name) %}
+
+{%- set exception_warning = "\n\nPlease be aware: The " ~ single_source_name|upper ~ "." ~ single_table_name|upper ~ " table was not found in your schema(s). The Fivetran Data Model will create a completely empty staging model as to not break downstream transformations. To turn off these warnings, set the `fivetran__remove_empty_table_warnings` variable to TRUE (see https://github.com/fivetran/dbt_fivetran_utils/tree/releases/v0.4.latest#union_data-source for details).\n"%}
+{%- set using_empty_table_warnings = (execute and not var('fivetran__remove_empty_table_warnings', false)) %}
+{%- set connections = var(connection_dictionary, []) %}
+{%- set using_unioning = connections | length > 0 %}
+
+{%- if using_unioning %}
+{# For unioning #}
+    {%- set relations = [] -%}
+    {%- for connection in connection_dictionary -%}
```

Comment:
```suggestion
    {%- for connection in connections -%}
```
Either we need to adjust this, or we need to revert [these](https://github.com/fivetran/dbt_github/commit/5efbf7ad43360fdd7dfbd6fcfcd0c3e6f8227020) changes for the `has_defined_sources` stuff to work
<img width="1522" height="552" alt="image" src="https://github.com/user-attachments/assets/8899b092-a632-45f0-9c87-80c5fe51bcd6" />

Reply:
oh whoops I did ctrl z too many times. updated.